djvm/src/main/kotlin/net/corda/djvm/analysis/AnalysisConfiguration.kt

```diff
@@ -127,9 +126,7 @@ class AnalysisConfiguration private constructor(
          * They should already exist within the sandbox namespace.
          */
         private val MANDATORY_PINNED_CLASSES: Set<String> = setOf(
```

Comment:
I am still a bit confused by the terminology: what is a PINNED and what is a  TEMPLATE  class?

Reply:
- A "pinned" class belongs to the application class loader, and the sandbox uses it "as is". All sandboxes share the same `class`.
- A "template" class belongs to the sandbox's class loader, which means that every sandbox has its own private definition of it. However, all these `class`es are created from identical bytecode which is read from the DJVM.